src/commands/money.js

```diff
@@ -49,6 +53,8 @@ export default class Money {
         const toQuery = command[2]
 
         if (toQuery == 'to' && !command[3]) {
+            
+            
```

Comment:
Line contains following spacing inconsistencies:
- Trailing whitespaces.

*Origin: SpaceConsistencyBear, Section: `all.whitespace`.*

The issue can be fixed by applying the following patch:

```diff
--- a/tmp/tmp49z0c0qn/src/commands/money.js
+++ b/tmp/tmp49z0c0qn/src/commands/money.js
@@ -54,7 +54,7 @@
 
         if (toQuery == 'to' && !command[3]) {
             
-            
+
             return this.VARIABLE.emptyTo
         }
 
```